modules/cluster-logging-kibana-visualize.adoc

```diff
@@ -59,3 +59,10 @@ For example:
 ----
 $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6k -- es_load_kibana_ui_objects <user-name>
 ----
+
+[NOTE]
+====
+The metadata of the kibana objects like visualizations, dashboards etc are stored in elastic search with the index format .kibana.{user_hash} where user_hash can be obtained by the command - userhash=$(echo -n $username | sha1sum | awk '{print $1}')
```

Comment:
elasticsearch

Reply:
if share_ops are used,  all users with cluster admin roles share the index. this metadata of the kibana objects is saved to .kibana index.

Reply:
Incoperated the changes. Thanks for the review comments.

Reply:
@anpingli @mijicherian One last question, I hope. Do we need to define or explain what share_ops is? It doesn't seem to be a term we use in the docs and I can find little information on the internet. Is share_ops a common concept?

Reply:
share_ops is enabled by default in 4.x.  I think mijicherian  had given a short instruction.

Reply:
@anpingli I found it, the `shared_ops` Kibana index mode. I changed the wording a bit to clarify. I am bolding the phrase here for clarity:
**If your cluster is configured to ue the Kibana *shared_ops* index mode (the default),** all users with cluster admin roles share the index and this Kibana object metadata is saved to the *.kibana* index.

See: https://github.com/openshift/openshift-docs/pull/18112/commits/433f0231ea928a746ccf25fa7ee7fc27dcef400b

Reply:
On second thought, we don't discuss the Kibana index mode in the docs in 4.x and @jcantrill suggested we keep it out for now. 
Should we remove the reference here to `shared_ops` to avoid confusion?